tests/unit/worker/test_build_dataset_worker.py

```diff
@@ -294,3 +303,15 @@ def test_delete_files_is_called(requests_mock, mocker):
 
     spy.assert_called()
 
+def _get_file_list_from_dir(directory: Path):
```

Comment:
This can be done with a list comprehension
```py
return [content for content in directory.iterdir() if content.is_file()]
```

Reply:
Clever and neat, done :)

Reply:
I see them now 👍🏻